plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraPartitionManager.java

```diff
@@ -82,7 +82,7 @@ public CassandraPartitionResult getPartitions(CassandraTableHandle cassandraTabl
                                 .flatMap(partitionTupleDomain -> partitionTupleDomain.getDomains()
                                         .map(Map::keySet)
                                         .map(Set::stream))
-                                .orElse(Stream.empty()))
+                                .orElseGet(Stream::empty))
```

Comment:
(X) constant

Reply:
in my JDK this is not a constant

```java
public static<T> Stream<T> empty() {
        return StreamSupport.stream(Spliterators.<T>emptySpliterator(), false);
    }
```

and then `StreamSupport.stream`:

```java
    public static <T> Stream<T> stream(Spliterator<T> spliterator, boolean parallel) {
        Objects.requireNonNull(spliterator);
        return new ReferencePipeline.Head<>(spliterator,
                                            StreamOpFlag.fromCharacteristics(spliterator),
                                            parallel);
    }
```

Reply:
my bad